src/frontend/components/SøknadsSteg/Dokumentasjon/LastOppVedlegg.tsx

```diff
@@ -83,55 +79,56 @@ const LastOppVedlegg: React.FC<Props> = ({ dokumentasjon, vedleggNr, oppdaterDok
         );
     };
 
+    const dokumentasjonsbeskrivelse = dokumentasjonsbehovTilBeskrivelseSanityApiNavn(
+        dokumentasjon.dokumentasjonsbehov
+    );
+
+    const vedleggXAvY = plainTekst(dokumentasjonstekster.vedleggXavY, {
+        antall: vedleggNr.toString(),
+        totaltAntall: antallVedlegg().toString(),
+    });
+    const vedleggtittel =
+        (dokumentasjon.dokumentasjonsbehov !== Dokumentasjonsbehov.ANNEN_DOKUMENTASJON
+            ? vedleggXAvY
+            : '') +
+        ' ' +
+        plainTekst(tittelBlock, { barnetsNavn: barnasNavn });
+
     return (
         <Container>
-            <Undertittel>
-                {dokumentasjon.dokumentasjonsbehov !== Dokumentasjonsbehov.ANNEN_DOKUMENTASJON && (
-                    <>
-                        <SpråkTekst
-                            id={'dokumentasjon.vedleggsnummer'}
-                            values={{
-                                vedleggsnummer: vedleggNr,
-                                antallvedlegg: antallVedlegg(),
-                            }}
-                        />
-                        &nbsp;
-                    </>
-                )}
-                {dokTittel}
-            </Undertittel>
-            {dokumentasjon.beskrivelseSpråkId && skalViseAnnenDokumentasjonsBeskrivelse() && (
-                <>
-                    <SpråkTekst
-                        id={dokumentasjon.beskrivelseSpråkId}
-                        values={{
-                            barn: formatertListeMedBarn(),
-                            meklingsattestLenke: (
-                                <EksternLenke
-                                    lenkeSpråkId={'dokumentasjon.meklingsattest.lenke'}
-                                    lenkeTekstSpråkId={'dokumentasjon.meklingsattest.lenketekst'}
-                                    target="_blank"
-                                />
-                            ),
-                        }}
-                    />
-                </>
+            <Heading level={'2'} size={'small'}>
+                {vedleggtittel}
+            </Heading>
+            {dokumentasjonsbeskrivelse && skalViseAnnenDokumentasjonsBeskrivelse() && (
+                <TekstBlock
+                    data-testid={'dokumentasjonsbeskrivelse'}
+                    block={dokumentasjonstekster[dokumentasjonsbeskrivelse]}
+                    typografi={Typografi.BodyLong}
+                />
             )}
             {!dokumentasjon.harSendtInn && (
-                <Filopplaster
-                    oppdaterDokumentasjon={oppdaterDokumentasjon}
-                    dokumentasjon={dokumentasjon}
-                    maxFilstørrelse={1024 * 1024 * 10}
-                    tillatteFiltyper={[EFiltyper.PNG, EFiltyper.PDF, EFiltyper.JPG, EFiltyper.JPEG]}
-                />
+                <div data-testid={'dokumentopplaster'}>
```

Comment:
Legger denne diven på noe ekstra styling? 🤔

Reply:
Måtte ha den for å få til test-id. Det funka ikke om man la det på komponenten inni :/

Reply:
Skjønner!